src/resources/views/components/dropdown/dropdown.blade.php

```diff
@@ -1,7 +1,9 @@
 @php($personalize = tallstackui_personalization('dropdown', $personalization()))
 
 <div @class($personalize['wrapper.first']) x-data="{ show : false, animate : @js(!$static) }">
```

Comment:
Please, try to add a logic to hide the dropdown when the position is `right` or `left` based on the mouse scroll.